packages/compiler/core/cli.ts

```diff
@@ -96,10 +96,15 @@ async function main() {
             string: true,
             describe: "Name of the emitters",
           })
+          .option("trace", {
+            type: "array",
+            string: true,
+            describe: "List of areas that should have the trace shown. e.g. `import-resolution.*`",
+          })
           .option("diagnostic-level", {
             type: "string",
-            default: "info",
-            choices: ["error", "warn", "info", "verbose", "debug"],
+            default: "trace",
+            choices: ["error", "warn", "trace"],
             describe: "Diagnostics of this level or above will be reported.",
```

Comment:
Hmm, this is weird to me. So if I do --diagnostic-level warn, then --trace effectively does nothing? It doesn't seem like that's helpful if so. I don't think we should conflate tracing with diagnostics in the public UX.

It feels like the only real use case is to disable warnings, and maybe --nowarn or something would be more discoverable than --diagnostic-level=error.

Reply:
Hmm, I think --nowarn is a bad name choice for disable all warnings as coming from csc, it is used with arguments to disable specific warnings.

I wonder if we can reuse the filtering for warnings, but inverted:

`--nowarn *` Disable all warnings
`--nowarn a.b.c` Disable a.b.c.* warnings

Reply:
yeah, wondering if we even want this

Reply:
I just removed it, I think we can revisit as a separate issue if we really want to mute warnings.